src/_pytest/helpconfig.py

```diff
@@ -170,6 +170,8 @@ def showhelp(config: Config) -> None:
         help, type, default = config._parser._inidict[name]
         if type is None:
             type = "string"
+        if help in (None, ""):
+            raise TypeError("help argument must be provided for {}".format(name))
```

Comment:
just to avoid confusion, can you have the exception explicitly state why it was raised? E.g.: `"help argument cannot be empty for {}".format(name)`.  Otherwise the user may feel like the exception is an error ( if they provided an empty string explicitly, for example )